src/ia-book-search-results.js

```diff
@@ -0,0 +1,104 @@
+import { html, LitElement } from 'lit-element';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+import bookSearchResultsCSS from './styles/ia-book-search-results.js';
+import closeIcon from './icon-close.js';
+
+export class IABookSearchResults extends LitElement {
+  static get styles() {
+    return bookSearchResultsCSS;
+  }
+
+  static get properties() {
+    return {
+      results: {
+        type: Array,
+      },
+    };
+  }
+
+  constructor() {
+    super();
+    this.results = [];
+    this.matchRegex = new RegExp('{{{(.+?)}}}', 'g');
+
+    this.bindBookReaderListeners();
+  }
+
+  bindBookReaderListeners() {
+    document.addEventListener('BookReader:SearchCallback', this.setResults.bind(this));
```

Comment:
Is using a document event listener for this for backwards compatibility or is this new logic? If this is new, is there another way we could communicate the results (ie callback or promise?).

Reply:
It's to provide compatibility with how BookReader currently triggers events. As part of removing jQuery from BookReader, we'll end up changing how these messages are passed out of BookReader.